docs/plugins/sponsorblock.md

```diff
@@ -0,0 +1,16 @@
+# SponsorBlock
+
+## What is SponsorBlock?
+SponsorBlock is an third party opensource extension and an API powered by community that skips automatically annoying segments in YouTube videos, like **paid sponsors**, **interaction reminders** ("Subscribe to my channel"), **resumes of previous videos**, **non-music parts** in music videos and more. It even has an option to directly go to the most important part of a video (Highlighted segment). You can customize which types of segments you want to be skipped and which you want to watch. This is integrated in **VueTube** as a plugin.
```

Comment:
- reword:  SponsorBlock is community**a** **crowdsourced** third-party **open source** extension and API
- grammar: that **automatically skips**
- grammar: annoying segments in YouTube video **such as** paid sponsors
- grammar: resumes of previous videos **and** non-music **sections*